packages/lit-dev-content/samples/tutorials/first-component/06/after/word-viewer.ts

```diff
@@ -0,0 +1,49 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+
+@customElement('word-viewer')
+class WordViewer extends LitElement {
+  static styles = css`
+    :host {
+      background-color: white;
+      color: violet;
+      cursor: pointer;
+      display: block;
+    }
+    pre {
+      padding: 0.2em;
+    }
+  `;
+
+  @state() private playDirection = 1;
```

Comment:
For the TS version, would it be worth to annotate this to a narrower type?
```suggestion
  @state() private playDirection: -1 | 1 = 1;
```

Reply:
Done!